xml/Microsoft.Extensions.Hosting.Internal/ConsoleLifetime.xml

```diff
@@ -1,287 +1,291 @@
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; ConsoleLifetime(Microsoft::Extensions::Options::IOptions&lt;Microsoft::Extensions::Hosting::ConsoleLifetimeOptions ^&gt; ^ options, Microsoft::Extensions::Hosting::IHostEnvironment ^ environment, Microsoft::Extensions::Hosting::IHostApplicationLifetime ^ applicationLifetime, Microsoft::Extensions::Options::IOptions&lt;Microsoft::Extensions::Hosting::HostOptions ^&gt; ^ hostOptions);" />
+      <MemberType>Constructor</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>Microsoft.Extensions.Hosting</AssemblyName>
+        <AssemblyVersion>3.0.0.0</AssemblyVersion>
+        <AssemblyVersion>3.1.0.0</AssemblyVersion>
+        <AssemblyVersion>5.0.0.0</AssemblyVersion>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+        <AssemblyVersion>7.0.0.0</AssemblyVersion>
+        <AssemblyVersion>8.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <Parameters>
+        <Parameter Name="options" Type="Microsoft.Extensions.Options.IOptions&lt;Microsoft.Extensions.Hosting.ConsoleLifetimeOptions&gt;" Index="0" FrameworkAlternate="dotnet-plat-ext-3.0;dotnet-plat-ext-3.1;dotnet-plat-ext-5.0;dotnet-plat-ext-6.0;dotnet-plat-ext-7.0;dotnet-plat-ext-8.0" />
+        <Parameter Name="environment" Type="Microsoft.Extensions.Hosting.IHostEnvironment" Index="1" FrameworkAlternate="dotnet-plat-ext-3.0;dotnet-plat-ext-3.1;dotnet-plat-ext-5.0;dotnet-plat-ext-6.0;dotnet-plat-ext-7.0;dotnet-plat-ext-8.0" />
+        <Parameter Name="applicationLifetime" Type="Microsoft.Extensions.Hosting.IHostApplicationLifetime" Index="2" FrameworkAlternate="dotnet-plat-ext-3.0;dotnet-plat-ext-3.1;dotnet-plat-ext-5.0;dotnet-plat-ext-6.0;dotnet-plat-ext-7.0;dotnet-plat-ext-8.0" />
+        <Parameter Name="hostOptions" Type="Microsoft.Extensions.Options.IOptions&lt;Microsoft.Extensions.Hosting.HostOptions&gt;" Index="3" FrameworkAlternate="dotnet-plat-ext-3.0;dotnet-plat-ext-3.1;dotnet-plat-ext-5.0;dotnet-plat-ext-6.0;dotnet-plat-ext-7.0;dotnet-plat-ext-8.0" />
+      </Parameters>
+      <Docs>
+        <param name="options">An object used to retrieve <see cref="T:Microsoft.Extensions.Hosting.ConsoleLifetimeOptions" /> instances.</param>
+        <param name="environment">An object that contains information about the hosting environment an application is running in.</param>
+        <param name="applicationLifetime">An object that allows consumers to be notified of application lifetime events.</param>
+        <param name="hostOptions">An object used to retrieve <see cref="P:Microsoft.Extensions.Hosting.Internal.ConsoleLifetime.HostOptions" /> instances.</param>

```

Comment:
@dotnet/area-extensions-hosting this API is internal (judging by the name). It's being flagged in the CI result. How should we adjust these docs to remove this mention?